showcase/pom.xml

```diff
@@ -88,10 +88,6 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-failsafe-plugin</artifactId>
-            <configuration>
-              <forkCount>1C</forkCount>
-              <reuseForks>true</reuseForks>
-            </configuration>
```

Comment:
Run the tests sequentially for now to see if it helps with the CI jobs.

Reply:
Can we revert this change with the fix?